src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -6525,6 +6525,19 @@ char *wm_vuldet_normalize_date(char **date) {
         }
     }
 
+    if (r_timestap) {
+        regfree(r_timestap);
```

Comment:
During the regex compilation by the function `regcomp` in lines `6466`, `6468` and `6470` some memory is allocated for the precompiled pattern buffer, and should be released with `regfree`.

Reply:
I see the same problem is happening in `wm_vuldet_get_term_condition`. Can you fix it too?